Bottom: 0b496b2661ca0965e8e2fd948c8210326e12953d
Top:    58a07bdcefd7c17fe83560ce0c04ac6fa3c1f5d7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-15 18:39:38 +0000

find.h: Annotate which exported functions are not used!


---

diff --git a/src/find.h b/src/find.h
index 9644727..7afe520 100644
--- a/src/find.h
+++ b/src/find.h
@@ -47,7 +47,7 @@ bool LineLineIntersect (LineType *, LineType *);
 bool LineArcIntersect (LineType *, ArcType *);
 bool PinLineIntersect (PinType *, LineType *);
 bool LinePadIntersect (LineType *, PadType *);
-bool ArcPadIntersect (ArcType *, PadType *);
+/* NOT USED EXTERNALLY */// bool ArcPadIntersect (ArcType *, PadType *);
 void LookupElementConnections (ElementType *, FILE *);
 void LookupConnectionsToAllElements (FILE *);
 void LookupConnection (Coord, Coord, bool, Coord, int, bool AndRats);
